lib/pg_rls/tenant.rb

```diff
@@ -18,10 +23,10 @@ def switch(resource)
       attr_reader :tenant
 
       def fetch
-        @fetch ||= tenant.find_by_tenant_id(
-          PgRls.connection_class.connection.execute(
-            "SELECT current_setting('rls.tenant_id')"
-          ).getvalue(0, 0)
+        @fetch ||= PgRls.main_model.send("find_by_tenant_id",
+                                         PgRls.connection_class.connection.execute(
+                                           "SELECT current_setting('rls.tenant_id')"
+                                         ).getvalue(0, 0)
```

Comment:
why `send`? instead of `find_by_tenant_id`?

Reply:
Good point! find_by_tenant_id would be simpler here!